### PR TITLE
reference/tidb-binlog: improve snapshot timestamp for initialization

### LINKED ISSUE #2080: TiDB binlog文档中的savepoint是什么意思?
File: [v3.0/reference/tidb-binlog/deploy.md](https://pingcap.com/docs-cn/v3.0/reference/tidb-binlog/deploy/)

>在已有的 TiDB 集群中启动 Drainer，一般需要全量备份并且获取 savepoint，然后导入全量备份，最后启动 Drainer 从 savepoint 开始同步增量数据。

这个`savepoint`在文档中并没有描述, 是指[这篇](https://pingcap.com/docs-cn/stable/reference/tidb-binlog/tidb-binlog-local/#%E6%B3%A8%E6%84%8F)的`-gen-savepoint`参数吗? 这应该已经是一个过期的参数和过期的文档吧?

在4.2 TiDB Binlog 安装部署实操 中是查看metadata的Pos值作为tso. 

这是不是属于文档内容缺失?, 说实话如果没有视频真是不知道怎么做
而视频中的mydumper备份命令其实有问题
```
--regex '^(?!(mysql|information_schema|performance_schema))'
```
应当为
```
--regex '^(?!(mysql\.|information_schema\.|performance_schema\.))'
```
详见https://github.com/maxbube/mydumper/issues/123

>既然是docs-cn , 希望回复用中文谢谢, 应该很少有外国人要看中文的TiDB文档并且来提Issue吧

### PR DESCRIPTION
**Pull request by july2993**
### What is changed, added or deleted? 
Improve about snapshot timestamp for init
fix #2080 

### Which version does your change affect? 

dev, v3.1, v3.0, v2.1

### CODE REVIEW

**Comment by july2993** [comment]
@suzaku PTAL

File: dev/reference/tidb-binlog/deploy.md [hunk 1]
```diff
@@ -597,6 +609,6 @@ Drainer="192.168.0.13"
 > - 在运行 TiDB 时，需要保证至少一个 Pump 正常运行。
 > - 通过给 TiDB 增加启动参数 `enable-binlog` 来开启 binlog 服务。尽量保证同一集群的所有 TiDB 都开启了 binlog 服务，否则在同步数据时可能会导致上下游数据不一致。如果要临时运行一个不开启 binlog 服务的 TiDB 实例，需要在 TiDB 的配置文件中设置 `run_ddl= false`。
 > - Drainer 不支持对 ignore schemas（在过滤列表中的 schemas）的 table 进行 rename DDL 操作。
-> - 在已有的 TiDB 集群中启动 Drainer，一般需要全量备份并且获取 savepoint，然后导入全量备份，最后启动 Drainer 从 savepoint 开始同步增量数据。
+> - 在已有的 TiDB 集群中启动 Drainer，一般需要全量备份并且获取 快照时间戳，然后导入全量备份，最后启动 Drainer 从对应快照时间戳开始同步增量数据。
```
Comment (suzaku) [inline dev/reference/tidb-binlog/deploy.md]:
```suggestion
> - 在已有的 TiDB 集群中启动 Drainer，一般需要全量备份并且获取*快照时间戳*，然后导入全量备份，最后启动 Drainer 从对应快照时间戳开始同步增量数据。
```

File: dev/reference/tidb-binlog/deploy.md [hunk 2]
```diff
@@ -148,6 +148,18 @@ Pump 和 Drainer 均可部署和运行在 Intel x86-64 架构的 64 位通用硬
 
     如果下游为 MySQL 或 TiDB，为了保证数据的完整性，需要进行全量数据的备份与恢复，必需使用全量备份的时间戳。
 
+    如果使用 mydumper 可以在导出目录的 metadata 文件的 Pos 字段获取对应时间戳, metadata 文件如下:
```
Comment (CaitinChen) [inline dev/reference/tidb-binlog/deploy.md]:
```suggestion
    如果使用 mydumper，可以在导出目录的 metadata 文件的 `Pos` 字段获取对应时间戳，metadata 文件如下:
```

File: dev/reference/tidb-binlog/deploy.md [hunk 3]
```diff
@@ -597,6 +609,6 @@ Drainer="192.168.0.13"
 > - 在运行 TiDB 时，需要保证至少一个 Pump 正常运行。
 > - 通过给 TiDB 增加启动参数 `enable-binlog` 来开启 binlog 服务。尽量保证同一集群的所有 TiDB 都开启了 binlog 服务，否则在同步数据时可能会导致上下游数据不一致。如果要临时运行一个不开启 binlog 服务的 TiDB 实例，需要在 TiDB 的配置文件中设置 `run_ddl= false`。
 > - Drainer 不支持对 ignore schemas（在过滤列表中的 schemas）的 table 进行 rename DDL 操作。
-> - 在已有的 TiDB 集群中启动 Drainer，一般需要全量备份并且获取 savepoint，然后导入全量备份，最后启动 Drainer 从 savepoint 开始同步增量数据。
+> - 在已有的 TiDB 集群中启动 Drainer，一般需要全量备份并且获取*快照时间戳*，然后导入全量备份，最后启动 Drainer 从对应快照时间戳开始同步增量数据。
```
Comment (CaitinChen) [inline dev/reference/tidb-binlog/deploy.md]:
```suggestion
> - 在已有的 TiDB 集群中启动 Drainer，一般需要全量备份并且获取**快照时间戳**，然后导入全量备份，最后启动 Drainer 从对应的快照时间戳开始同步增量数据。
```

Comment (CaitinChen) [reply dev/reference/tidb-binlog/deploy.md]: Italic fonts don't apply to Chinese characters. :-)

**Comment by CaitinChen** [comment]
@july2993 Please address comments. Thanks.

**Comment by lilin90** [comment]
@CaitinChen PTAL again~ Thanks!

**Comment by CaitinChen** [comment]
@july2993 LGTM. You can update other versions now.

**Comment by july2993** [comment]
> @july2993 LGTM. You can update other versions now.

done

File: dev/reference/tidb-binlog/deploy.md [hunk 4]
```diff
@@ -148,6 +148,18 @@ Pump 和 Drainer 均可部署和运行在 Intel x86-64 架构的 64 位通用硬
 
     如果下游为 MySQL 或 TiDB，为了保证数据的完整性，需要进行全量数据的备份与恢复，必需使用全量备份的时间戳。
 
+    如果使用 mydumper，可以在导出目录的 metadata 文件的 `Pos` 字段获取对应时间戳，metadata 文件如下:
```
Comment (lilin90) [inline dev/reference/tidb-binlog/deploy.md]:
```suggestion
    如果使用 mydumper，可以在导出目录的 metadata 文件的 `Pos` 字段获取对应时间戳，metadata 文件如下：
```

File: v2.1/reference/tidb-binlog/deploy.md [hunk 5]
```diff
@@ -128,6 +128,24 @@ Pump 和 Drainer 均可部署和运行在 Intel x86-64 架构的 64 位通用硬
 
 1. 获取 initial_commit_ts
 
+    如果从最近的时间点开始同步，可以参考下文使用 binlogctl 获取一个最新的时间戳。
+
+    如果下游为 MySQL 或 TiDB，为了保证数据的完整性，需要进行全量数据的备份与恢复，必需使用全量备份的时间戳。
+
+    如果使用 mydumper，可以在导出目录的 metadata 文件的 `Pos` 字段获取对应时间戳，metadata 文件如下:
```
Comment (lilin90) [inline v2.1/reference/tidb-binlog/deploy.md]:
```suggestion
    如果使用 mydumper，可以在导出目录的 metadata 文件的 `Pos` 字段获取对应时间戳，metadata 文件如下：
```

File: v3.0/reference/tidb-binlog/deploy.md [hunk 6]
```diff
@@ -148,6 +148,18 @@ Pump 和 Drainer 均可部署和运行在 Intel x86-64 架构的 64 位通用硬
 
     如果下游为 MySQL 或 TiDB，为了保证数据的完整性，需要进行全量数据的备份与恢复，必需使用全量备份的时间戳。
 
+    如果使用 mydumper，可以在导出目录的 metadata 文件的 `Pos` 字段获取对应时间戳，metadata 文件如下:
```
Comment (lilin90) [inline v3.0/reference/tidb-binlog/deploy.md]:
```suggestion
    如果使用 mydumper，可以在导出目录的 metadata 文件的 `Pos` 字段获取对应时间戳，metadata 文件如下：
```

File: v3.1/reference/tidb-binlog/deploy.md [hunk 7]
```diff
@@ -147,6 +147,18 @@ Pump 和 Drainer 均可部署和运行在 Intel x86-64 架构的 64 位通用硬
 
     如果下游为 MySQL 或 TiDB，为了保证数据的完整性，需要进行全量数据的备份与恢复，必需使用全量备份的时间戳。
 
+    如果使用 mydumper，可以在导出目录的 metadata 文件的 `Pos` 字段获取对应时间戳，metadata 文件如下:
```
Comment (lilin90) [inline v3.1/reference/tidb-binlog/deploy.md]:
```suggestion
    如果使用 mydumper，可以在导出目录的 metadata 文件的 `Pos` 字段获取对应时间戳，metadata 文件如下：
```

**Comment by lilin90** [comment]
Rest LGTM

**Pull request merged**